lib/Tomcat/Utils.pm

```diff
@@ -49,8 +50,10 @@ EOF
     select_console('root-console');
 
     record_info('Initial Setup');
+    # we need to disable packagekit because it can block zypper sometimes
+    quit_packagekit;
```

Comment:
That should not be used?

At least on TW (where I checked) we run on a generated image that disables/masks packagekit as part of system_prepare (https://openqa.opensuse.org/tests/3107267#step/system_prepare/20) - so after booting that image, the service Is masked and should not be running

Reply:
I hit this on sles :)